packages/core/src/App/Components/Elements/LiveChat/live-chat.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Popover, Icon, Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import useLiveChat from 'App/Components/Elements/LiveChat/use-livechat';
+
+type TLiveChatProps = { is_mobile_drawer?: boolean };
+
+const LiveChat = observer(({ is_mobile_drawer }: TLiveChatProps) => {
+    const { client } = useStore();
+    const { has_cookie_account, loginid } = client;
+    const liveChat = useLiveChat(has_cookie_account, loginid);
+
+    if (!liveChat.isReady) return null;
+
+    const liveChatClickHandler = () => liveChat.widget?.call('maximize');
+
+    if (is_mobile_drawer)
```

Comment:
Since `is_mobile_drawer` is used to indicate that the component is to be rendered during mobile view, can we remove this prop and use `MobileWrapper` and `DesktopWrapper` or  `is_mobile` and `is_dektop`?
That way u needn't pass additional props